{"name": "Wrong user password error test", "status": "passed", "start": 1732715168990, "stop": 1732715172228, "uuid": "114a33d0-2693-467b-87a5-7fb2f0b50611", "historyId": "1e6ee410fb0ff0642fa49ffd43caad78", "testCaseId": "1e6ee410fb0ff0642fa49ffd43caad78", "fullName": "tests.test_login_user.TestUserLogin#test_user_login_with_wrong_password", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_user"}, {"name": "subSuite", "value": "TestUserLogin"}, {"name": "host", "value": "ast24help1c2"}, {"name": "thread", "value": "11252-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_user"}]}